docs/repo-guides/release-nnf-sw/readme.md

```diff
@@ -89,93 +91,104 @@ just an example.
     1. For `lustre-csi-driver`, `lustre-fs-operator`, `dws`, `nnf-sos`, and `nnf-dm` there are additional files that need to
     track the version number as well, which allow them to be installed with `kubectl apply -k`.
 
-        1. For `lustre-fs-operator`, `dws`, `nnf-sos`, and `nnf-dm` update `config/manager/kustomization.yaml` with the correct version.
-
-        2. For `lustre-csi-driver`, update `deploy/kubernetes/base/kustomization.yaml` and
-        `charts/lustre-csi-driver/values.yaml` with the correct version.
-
-    2. If `nnf-mfu` was updated, multiple references in `nnf-dm` will need to be updated with the
-    new version number for the `nnf-mfu` image:
-
-        1. In `Dockerfile` and `Makefile`, replace `NNFMU_VERSION` with the new version
+    |Repo                 |Update|
+    |---------------------|------|
+    |`nnf-mfu`            |The new version of `nnf-mfu` is referenced by the `NNFMFU` variable in several places:<br><br>`nnf-sos`<br>1. `Makefile` replace `NNFMFU` with `nnf-mfu's` tag<br><br>`nnf-dm`<br>1. In `Dockerfile` and `Makefile`, replace `NNFMU_VERSION` with the new version<br>2. In `config/manager/kustomization.yaml`, replace `nnf-mfu`'s `newTag: <X.Y.Z>`|
```

Comment:
The rendered version of this looked odd with the 'nnf-mfu' on the left and centered vertically.  Makes perfect sense to me now, but it took a while for me work out what was going on.


This is a collection of sentences.  So let's get periods on the end of each one.  That'll help the eyes sort things out.